google/cloud/bigquery/v2/minimal/internal/job_idempotency_policy.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Internal interface for Bigquery V2 Job resource.
```

Comment:
I am not sure we want to reference internal source files here.  We can, but seems a bit odd.  And generally we document the C++ symbols more than the files.